stl/inc/algorithm

```diff
@@ -10507,6 +10507,29 @@ namespace ranges {
 #endif // __cpp_lib_concepts
 #endif // _HAS_CXX17
 
+#if _HAS_CXX20
+#ifdef __cpp_lib_concepts
+template <class InputIterator1, class InputIterator2, class Cmp>
+constexpr auto lexicographical_compare_three_way(InputIterator1 b1, InputIterator1 e1, InputIterator2 b2,

```

Comment:
This is already implemented in `main` (and has merged to `feature/spaceship`); 
https://github.com/microsoft/STL/blob/47cd70315d560e4fe1ac699c0eb8ec8bfc5cb658/stl/inc/xutility#L4844-L4846
https://github.com/microsoft/STL/blob/47cd70315d560e4fe1ac699c0eb8ec8bfc5cb658/stl/inc/xutility#L4895-L4899

In fact, there appears to be no remaining work for the Algorithms clause (which I didn't realize when adding cards to the [Spaceship project](https://github.com/microsoft/STL/projects/4)). All of the [wording changes](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1614r2.html#clause-25-algorithms-library) are present in `main`, including the issue [LWG-3350](https://cplusplus.github.io/LWG/issue3350) "Simplify return type of `lexicographical_compare_three_way`" that was noted in the project card. IIRC, this work was done when @CaseyCarter implemented the original Spaceship paper [P0768R1](https://wg21.link/P0768R1) and was aware that the renames in [P1614R2](https://wg21.link/P1614R2) Adding Spaceship `<=>` To The Library were coming.

I'll go update the Spaceship project accordingly - my apologies for not correctly recording the status earlier.